lib/KBox/src/KMessage.h

```diff
@@ -191,6 +196,94 @@ class IMUMessage: public KMessage {
     };
 };
 
+class NAVMessage: public KMessage {
+  private:
+    bool apMode, apHeadingMode, apWaypointMode, apDodgeMode;
+    double currentHeading, targetHeading, courseToWaypoint;
```

Comment:
Can you help me understand what is DODGE MODE? 

Is it a mode where we stop controlling the motors for X seconds so that the helmsman can dodge something else and then quickly get back to the previous setting without having to re-set the right heading?